core/src/com/unciv/ui/screens/cityscreen/CityScreen.kt

```diff
@@ -60,8 +61,19 @@ class CityScreen(
         const val wltkIconSize = 40f
     }
 
+    private val selectedCiv: Civilization = GUI.getWorldScreen().selectedCiv
+
+    private val isSpying = selectedCiv.gameInfo.isEspionageEnabled() && selectedCiv != city.civ
```

Comment:
What happens if we are a different civ, but espionage is not enabled...?

Reply:
The default, they won't be seeing the city screen in the first place. foreignCityInfoPopup() will be called instead.

<details><summary>Picture</summary>

![ForignCityView](https://github.com/yairm210/Unciv/assets/7538725/bf230072-c52b-40e9-addd-53a0acd91df7)


</details>